ui/ozone/platform/wayland/fake_server.cc

```diff
@@ -15,6 +15,10 @@
 #include "base/run_loop.h"
 #include "base/strings/string_number_conversions.h"
 
+#if BUILDFLAG(USE_XKBCOMMON)
```

Comment:
Maybe you better move the MockWaylandXkbKeyboardLayoutEngine to a separate source file and avoid adding checks for the USE_XKBCOMMON build flags all around here?

And all the mocked classes are inheritted from the ServerObject base class and kinda behave as server objects. MockWaylandXkbKeyboardLayoutEngine doesn't really suit this pattern much.

Reply:
that is a good point. Done.